scripts/paxcheck.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/sh
 
-if ! type scanelf > /dev/null 2>&1; then
+SCANELF="/usr/bin/scanelf"
```

Comment:
We can't safely assume paths.  Plus `dash`, `ash`, and `bash` all support type.  Let's disable the warning.